src/content/integrations/@unocssastro.md

```diff
@@ -8,4 +8,6 @@ npmUrl: "https://www.npmjs.com/package/@unocss/astro"
 repoUrl: "https://github.com/unocss/unocss"
 homepageUrl: "https://github.com/unocss/unocss#readme"
 downloads: 975889
+# unocss bundles its Astro integration with all installs, real usage is much lower than the download count suggests
+downloadFactor: 0.02
```

Comment:
So this is just an arbitrary number to lower inflated download counts? I feel like this could open up a can of worms, but I can't think of anything better

Reply:
In case it’s any reassurance, it’s not 100% arbitrary — I calculated this based on some telemetry data to scale it to the rough ballpark. But yeah, obviously a bit of a weird one. Luckily there only seemed to be a couple of these where the parent package is so popular that it caused issues.